types/testingutils/beacon_node.go

```diff
@@ -2,122 +2,137 @@ package testingutils
+	var res capella.ExecutionPayload
 	if err := yaml.Unmarshal(data, &res); err != nil {
 		panic(err.Error())
 	}
-	return res.Transactions
+	return res
+}
+
+var Transactions = func() []bellatrix.Transaction {
```

Comment:
don't we have `capella.Transaction`? or this is a mistake?

Reply:
no, the transactions introduced in Bellatrix fork